### PR TITLE
UI Changes + Linter warning fixes

### PR DESCRIPTION
**Pull request by brionmario**
This PR introduces the following UI enhancements to the user portal.

#### 1.  User Avatar

Add support to generate initials with first name and last name.

Screen Shot 2019-10-23 at 2 29 06 PM

Screen Shot 2019-10-23 at 2 28 54 PM

Extend gravatar support to the header dropdown.

Screen Shot 2019-10-23 at 2 34 03 PM

#### 2. Approvals

Revamp pending approvals UI.

**Before**

Screen Shot 2019-10-23 at 2 22 59 PM

**After**

Screen Shot 2019-10-23 at 2 29 19 PM

#### 3. Sections

Add support to include an icon in the primary action.

Screen Shot 2019-10-23 at 4 47 38 PM

### CODE REVIEW

File: modules/theme/src/themes/default/views/card.overrides [hunk 1]
```diff
@@ -36,7 +39,20 @@
             }
         }
         .top-action-panel {
-            padding: 0 @settingsCardPadding;;
+            box-shadow: none;
+            padding: 0 @settingsCardPadding;
+            border-left: 0px;
+            border-right: 0px;
+            border-top: 1px solid #eee;
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/views/card.overrides]: Shall we get this value from a variable?

File: modules/theme/src/themes/default/views/card.overrides [hunk 2]
```diff
@@ -36,7 +39,20 @@
...
+            border-bottom: 1px solid #eee;
+
+            .description {
+                display: flex;
+                align-items: center;
+            }
+            &.no-margin-bottom {
+                margin-bottom: 0px;
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/views/card.overrides]: Remove `px` when have `0` value

File: modules/theme/src/themes/default/views/card.overrides [hunk 3]
```diff
@@ -53,6 +69,9 @@
                         padding-right: @settingsCardPadding;
                     }
                 }
+                &.no-margin-top {
+                    margin-top: 0px;
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/views/card.overrides]: Remove `px` when have `0` value

File: modules/theme/src/themes/default/views/card.overrides (hunk 1 above)
Comment (brionmario) [reply modules/theme/src/themes/default/views/card.overrides]: This value was moved to the variable `settingsCardTopActionPanelBorderTop`

**Pull request merged**